lib/Db/MessageMapper.php

```diff
@@ -560,6 +564,10 @@ public function findIdsByQuery(Mailbox $mailbox, SearchQuery $query, ?int $limit
 			}
 		}
 
+		if (isset($flags[Horde_Imap_Client::FLAG_MDNSENT])) {
+			$select->andWhere($qb->expr()->eq('flag_mdnsent', $qb->createNamedParameter('flag_mdnsent', IQueryBuilder::PARAM_BOOL)));
```

Comment:
can we add this flag to the array above at L550-558?

Reply:
If it's just about the leading `$` then we can maybe add `'mdnsent'`, right?

Reply:
https://github.com/nextcloud/mail/pull/4343/commits/f2a19f96caeb79e8551014febb659efb1b8e7940